docs/37-actions.mdx

```diff
@@ -0,0 +1,67 @@
+
+## Object Toggle
+`Turn On` turns the specified Object on, and sets the default state of the object to off, `Turn Off` does the opposite.\
+`Flip State (Deprecated)` simply flips the current state the object of in the editor. You may see this on toggles created in older versions of VRCFury.
+
+## BlendShape
+Animates a Blendshape to the defined value. By default, the Blendshape will be animated on all renderers/meshes of the avatar, you can also uncheck this and define certain meshes it should apply to.
+
+## Animation Clip
+Runs a provided pre-made animation clip. Useful if you want to animate something not supported by VRCFury Actions yet or non-static animations, such as a slider animating multiple blendshapes at different points. If `Loop Time` is checked on the provided animation clip, the animation will loop.
+:::tip
+When used in a Toggle, Gesture or other VRCFury component, VRCFury will first try to match the animation path relative to the object said VRCFury component is placed on. This means that if the object you are animating is a child of that component, you can move it anywhere on the avatar and the path will be adjusted automatically.\
+This is similar to how a [Full Controller](/components/full-controller) works.
+:::
+
+## Poiyomi Flipbook Frame
+Animates the current frame of a Poiyomi Flipbook on the specified renderer to the defined value.
+
+## Poiyomi UV Tile
+Sets up toggles for [Poiyomi UV Tile Discard](https://www.poiyomi.com/special-fx/uv-tile-discard).
+
+## SCSS Shader Inventory
+Similar to the Poiyomi UV Tile action, but for [Silent's SCSS](https://github.com/s-ilent/scss)
+
+## Material Property
+Animates a Material Property on the specified renderer to the defined value. In many shaders, you can simply right click a slider or value to copy its name. If your shader supports locking, make sure to mark the Material Property you are animating as animated.
+If the Material Property should be applied to all renderers on the avatar, enable `Apply to all renderers`. This is useful for avatar-wide toggles such as Audiolink or lighting modifications\
+Note: Material Properties will be applied across the entire renderer, this is a unity limitation. If your shader has any, This can be avoided by using the Renaming function of your shader, for example [Poiyomi Rename Animated](https://www.poiyomi.com/general/locking#rename-animated).
+
+## Scale
+Scales the specified object to the defined multiplier. Note that this is not the absolute scale value, but rather a multiplier. For example, an object set to an absolute scale of 0.75 will be animated to ab absolute scale of 1.5 by a Scale action set to a multipler of 2.
+
+## Material Swap
+Swaps the material on the specified renderer and material slot to the defined material. Note that unity starts counting material slots at 0. If you are unsure of the slot number you want to animate, look at the Element number shown in the `Skinned Mesh Renderer` component on the mesh.
```

Comment:
better to say `Renderer` since this also works on mesh renderers.

Reply:
Changed